docs/local-dind-tutorial.md

```diff
@@ -85,17 +86,17 @@ demo-tidb-peer    ClusterIP   None             <none>        10080/TCP
 demo-tikv-peer    ClusterIP   None             <none>        20160/TCP                        1m
 
 $ kubectl get configmap -n tidb
```

Comment:
The output needs to be adjusted because of #435

Reply:
The output of configmap has no difference with the one currently in doc.